source/common/http/filter/lua/lua_filter.cc

```diff
@@ -261,8 +261,22 @@ int StreamHandleWrapper::luaHeaders(lua_State* state) {
   if (headers_wrapper_.get() != nullptr) {
     headers_wrapper_.pushStack();
   } else {
-    headers_wrapper_.reset(
-        HeaderMapWrapper::create(state, headers_, [this]() { return !headers_continued_; }), true);
+    headers_wrapper_.reset(HeaderMapWrapper::create(state, headers_,
+                                                    [this] {
+                                                      // If we are about to do a modifiable header
```

Comment:
Nit: How come clang-format makes so much indent, so much wow, here?

Reply:
Yeah I don't know. It bothered me also but not sure what I can do about it. It seems like clang-format bug.

Reply:
OK, no worries.